### DISCUSSION TITLE
Is OPOS 3.4 working or still unasable ?

### DISCUSSION THREAD

**Discussion by Giuliano69** [opensourcepos/opensourcepos#4087] [Q&A]
I'm installing OPOS over ubuntu 2024.04,  that comes with PHP 8.3
PHP 7.4 reched EOL  in 2022... it has no support nor more fixes. IT may be a security issue...

Installing/compiling the 3.4 (with php8.3) gives an usable software, or it is buggy and unusable ?

The goal, is to use it in a shop, installing/configuring/managing it by ourself...
What threaten me from using the 3.3.9 is:
- php 7.4 is in EOL form two years ago....security reason for not to install php 7.4
- there could be a big database change and data migration from 3.3.9 ti 3.4

So IF the 3.4 is working, I would prefer installing it...

What do you suggest ?

BR
Giuliano

**Comment by ekardian** [comment]
wait son for the new stable

**Comment by jekkos** [comment]
A build is available now as a prerelease in https://github.com/opensourcepos/opensourcepos/releases It will be refreshed for every commit merged to master
